components/channel_header/channel_header.js

```diff
@@ -417,19 +421,35 @@ class ChannelHeader extends React.PureComponent {
 
         let dmHeaderIconStatus;
         let dmHeaderTextStatus;
+        let dmHeaderCustomStatus;
         if (isDirect && !dmUser.delete_at && !dmUser.is_bot) {
             dmHeaderIconStatus = (
                 <StatusIcon
                     status={channel.status}
                 />
             );
 
+            dmHeaderCustomStatus = this.props.isCustomStatusEnabled && (
+                <>
+                    <CustomStatusEmoji
+                        userID={this.props.dmUser.id}
+                        emojiSize={15}
+                        emojiStyle={{
+                            verticalAlign: 'top',
+                            margin: '0px 4px',
```

Comment:
No need to specify dimensions for `0`

```suggestion
                            margin: '0 4px',
```